docs/wiki/Cryptography/bls.md

```diff
@@ -0,0 +1,232 @@
+<!-- @format -->
+
+# Boneh-Lynn-Shacham (BLS) signature
+
+### TLDR;
+
+- Proof of stake protocols use digital signatures to identify their participants and hold them accountable.
+- BLS signatures can be aggregated together, making them efficient to verify at large scale.
+- Signature aggregation allows the beacon chain to scale to hundreds of thousands of validators.
+- Ethereum transaction signatures on the execution (Eth1) layer remain as-is.
+
+BLS is a digital signature scheme with aggregation properties. Given set of signatures (_signature_1_, ..., _signature_n_) anyone can produce an aggregated signature. Aggregation can also be done on secret keys and public keys. Furthermore, the BLS signature scheme is deterministic, non-malleable, and efficient. Its simplicity and cryptographic properties allows it to be useful in a variety of use-cases, specifically when minimal storage space or bandwidth are required. This page will cover general idea and Math behind BLS signatures, further cover BLS in context of Ethereum.
+
+## How BLS works?
+
+At the core of BLS signatures is the concept of bilinear mapping through pairings on elliptic curves. A key component is a pairing function $e$, defined between two groups derived from elliptic curves:
+
+$$e: G_1 \times G_2 \rightarrow G_T$$
+
+This function is efficiently computable and must satisfy bilinear properties:
+
+- For all $P,Q$ in $G_1$ and $a$ in integers, bilinearity is defined as:
+  $$ e(aP, Q) = e(P, Q)^a $$
+  $$ e(P, aQ) = e(P, Q)^a $$
+
+- Additionally, it must distribute over addition:
+  $$ e(P + Q, R) = e(P, R) \times e(Q, R) $$
+  $$ e(P, Q + R) = e(P, Q) \times e(P, R) $$
+
+These properties enable the cryptographic mechanisms necessary for functions like signature aggregation, which is a pivotal feature in blockchain applications and cryptographic consensus.
+
+#### Transition from ECDSA to BLS
```

Comment:
Maybe better title would be a comparison to ecdsa?

Reply:
replaced it with something better